app/src/main/res/layout/activity_home.xml

```diff
@@ -24,6 +24,11 @@
     </ScrollView>
 
 
+    <include
```

Comment:
move the content of `app_bar_main` here instead of `include` (we are not going to reuse it, so no use of a new file)